xmtp_mls/src/storage/encrypted_store/mod.rs

```diff
@@ -587,6 +591,83 @@ pub(crate) mod tests {
         EncryptedMessageStore::remove_db_files(db_path)
     }
 
+    #[tokio::test]
```

Comment:
Test won't run in WASM without the `#[wasm_bindgen_test::wasm_bindgen_test(unsupported = tokio::test)]